Client.xcodeproj/project.pbxproj

```diff
@@ -8268,44 +8268,44 @@
 			shellPath = /bin/sh;
 			shellScript = "if [ $ACTION != \\\"indexbuild\\\" ]; then\n   bash $PWD/bin/nimbus-fml.sh -n MozillaAppServices\nfi\n";
 		};
-		C874A4E327F62C5B006F54E5 /* Swiftlint */ = {
```

Comment:
@isabelrios 
I'm not sure why it generated those changes inside the project.pbxproj 🤔 I don't think this should be there